src/js/services/ontotext-yasgui-web-component.service.js

```diff
@@ -0,0 +1,38 @@
+const modules = [];
+
+angular
+    .module('graphdb.framework.ontotext-yasgui-web-component', modules)
+    .factory('OntotextYasguiWebComponentService', OntotextYasguiWebComponentService);
+
+OntotextYasguiWebComponentService.$inject = ['MonitoringRestService', 'RDF4JRepositoriesRestService', '$repositories', 'toastr', '$translate'];
+
+function OntotextYasguiWebComponentService(MonitoringRestService, RDF4JRepositoriesRestService, $repositories, toastr, $translate) {
```

Comment:
hmm it was a good idea for this service. Maybe not the best name for it, but we will probably definetly need a dedicated service for the sparql editor.

Reply:
Yes, you are right about the name. I named it that way so we don't confuse our colleagues when we start merging into master. When we clean up the old yasgui code, we can rename it to a simpler name "YasguiService" or something similar.